src/js/utils/backbone.events.ts

```diff
@@ -166,14 +198,15 @@ const eventSplitter = /\s+/;
 // Implement fancy features of the Events API such as multiple event
 // names `"change blur"` and jQuery-style event maps `{change: action}`
 // in terms of the existing API.
-function eventsApi(obj, action, name, rest) {
+function eventsApi(obj: Events, action: EventsAction, name: string | EventListenerDictionary | undefined, rest: any[]): name is string {
```

Comment:
`name?: string | EventListenerDictionary`?

Reply:
`name` is optional in `off`. So if you call `object.off()`, then `eventsApi(this, 'off', undefined, [undefined, undefined])` is invoked.